sample.env

```diff
@@ -0,0 +1,12 @@
+ENV=0
```

Comment:
Don't commit this file

Reply:
It's just a sample file. Khaja asked me to commit it